deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/graph/ComputationGraph.java

```diff
@@ -318,6 +319,21 @@ public void setLabels(INDArray... labels) {
     }
 
     /**
+     * This method allows you to specificy GradientsAccumulator instance to be used with this model
+     *
+     * PLEASE NOTE: Do not use this method unless you understand how to use GradientsAccumulator & updates sharing.
+     * PLEASE NOTE: Do not use this method on standalone model
+     *
+     * @param accumulator
+     */
+    public void setGradientsAccumulator(GradientsAccumulator accumulator) {
```

Comment:
Random thought with these sorts of methods: should we introduce our own annotations (probably defined in ND4J and used everythere) for these sorts of things?
Similar to what Spark does: https://spark.apache.org/docs/1.5.2/api/java/org/apache/spark/annotation/package-summary.html

It'd let people know, and (if easy/possible) it should automatically add warnings to (html/rendered) javadoc etc.